fuzzers/qemu_launcher/Makefile.toml

```diff
@@ -1,86 +1,250 @@
-# Variables
 [env]
-FUZZER_NAME='libpng_harness'
-PROJECT_DIR = { script = ["pwd"] }
+CROSS_CC = "x86_64-linux-gnu-gcc"
+CROSS_CXX = "x86_64-linux-gnu-g++"
+CROSS_CFLAGS = ""
+TARGET_DIR = "${CARGO_MAKE_CRATE_TARGET_DIRECTORY}/x86_64"
+LIBPNG_ARCH = "x86_64"
+LIBPNG_OPTIMIZATIONS = "yes"
+FEATURE = "x86_64"
+#LIBAFL_DEBUG_OUTPUT = "1"
+#CUSTOM_QEMU_DIR= "~/qemu-libafl-bridge"
+
+[env.arm]
+CROSS_CC = "arm-linux-gnueabi-gcc"
```

Comment:
I guess we could try to figure all of these out in a build.rs in the bindings, right?

Reply:
Not sure I follow? Can you elaborate please?

Reply:
Like, in the bindings, we could find if there is one of a few well-known cross toolchains available. That we users don't need to set this amount of env variables every time.
It's more a general idea, not related to the current PR

Reply:
The bindings will detect the toolchain based on the host architecture rather than the guest though? So we would need to find the implementation and modify it to worth depending on the guest architecture we set using the features?

Reply:
Yes exactly, the feature flag could be checked in build.rs and then we can try to find the correct cross toolchain